app/controllers/admin/base_controller.rb

```diff
@@ -58,7 +58,11 @@ def fetch_counts
       @admins = User.agents #can_receive_ticket.tagged_with(current_user.team_list, :any => true)
     else
       topics = Topic.all
-      @admins = User.agents
+      first = [current_user]
+      @admins = [*first, *(User.agents - first)]
+      #@admins = User.agents
+      #first = [current_user]
+      #puts [*first, *(@admins.sort - first)].inspect
```

Comment:
ok i tried. i guess im just a popular target, eh? isn't this coding stuff supposed to be for good? why can't you pick on someone else...i have no money, although you are costing me money, and time, in which i have very little of. can everyone just leave me alone? please? i have asked nicely....even said i'd stay on my side and you could do your thing, but not to interfere with my livelyhood..which you have. i shouldn't have to get a another new phone, or take back my fire stick, or not be able to get or send emails to my lawyer regarding my case. you know i've been diagnosed and am dying...why you wanna pick on me? and when my son couldn't watch kong when he visited me that was it. i am finding out all the worms up my rear. yes, i've contacted the police and fbi but after finding out who all the players are, well i think i have, i was kind of hoping i wouldn't need to go down that road. i am only one sick old lady with a lot of limited time on my hands. granted, i am learning and going to continue to learn, but this is illegal. it's fraud and identity theft from what i was told by the police. i imagine if i go down to the station and they do assign someone they will know what to do. i can't afford a new laptop and this internet service doesn't belong to me...and yet you take it. i agree...internet should be free, which is why i will be getting my own vpn soon. i really don't want to scratch the emails that i have had for 10-15 years, which i think that's what hackers get...if they are good anyway. i don't know how good you are, you're better than me, but i'm trying. so with the billions of people in the world much more interesting than me, why not them? you guys have ruined my good standing with a few websites and that makes me pretty upset as well. not like it does about my autistic son not being able to see a movie i promised though..that's what got me on a mission. you ever deal with an autistic child who doesn't get what he is promised? it isn't pretty. listen...i have got so much on my plate right now and would like to be doing other things instead of chasing down ghosts...now one's with names at least. i want things put back the way they were found. there's a challenge for you. see how fast you can do that. my settings are supposed to belong to ME. and i want my phone un-gmail-locked please.